packages/Panel/src/components/Dialog/Dialog.js

```diff
@@ -48,7 +48,6 @@ function Dialog(props) {
       </sc.DialogContent>
     </>
   );
-
   React.useEffect(() => {
     setIsAnimating(true);
   }, [isOpen]);
```

Comment:
Do we still need this effect then?

Reply:
Well it gets called on mount, and any time the `isOpen` prop changes, which comes from the Panel state, which changes whenever you open or close it, so without doing any larger refactor I would say yes